lib/kvdb/ikvdb.c

```diff
@@ -389,20 +458,42 @@ ikvdb_storage_add(const char *kvdb_home, struct kvdb_cparams *params)
 
     if (meta.km_version != KVDB_META_VERSION || meta.km_omf_version != GLOBAL_OMF_VERSION) {
         err = merr(EPROTO);
-        log_errx("cannot add storage to kvdb %s, out-of-date meta/on-media versions %u/%u: @@e",
+        log_errx("cannot add storage to kvdb (%s), out-of-date meta/on-media versions %u/%u: @@e",
                  err, kvdb_home, meta.km_version, meta.km_omf_version);
```

Comment:
We need a generic message for:  "Please upgrade to the latest version of HSE.  KVDB '%s' was created with a newer version of HSE and is not compatible with version %s." Can be another PR since it probably touches other areas of code.  Maybe a good use of hse_err_ctx!

Reply:
Yep, I will include this in my next PR.